cypress/common/submit-application.js

```diff
@@ -0,0 +1,67 @@
+import { getRandomApplicantAge } from './birthdate-constants';
```

Comment:
Helper file to submit an application with a couple basic params, keeping things dry.

Reply:
👍